backend/development.ini.sample

```diff
@@ -178,12 +179,30 @@ workspace.join.max_messages_history_count = -1
 ; sqlalchemy.url = sqlite:///%(here)s/tracim.sqlite
 sqlalchemy.url = %(basic_setup.sqlalchemy_url)s
 
-### File Depot ###
-## file depot name
-; depot_storage_name = tracim
-## This is where none database file will be stored
-; depot_storage_dir = %(here)s/depot
-depot_storage_dir = %(basic_setup.depot_storage_dir)s
+### Uploaded files ###
+
+; uploaded_files.storage_name = tracim
+## storage type, can be either "local" (local dir), "memory" (ram storage) or "s3" (remote S3 compatible storage)
```

Comment:
Why use `"memory" (ram storage)` and not directly `ram` ? (question visible also in issue)

Reply:
https://github.com/tracim/tracim/issues/3908#issuecomment-741703838

Reply:
`memory` is what other projects use for this, I think we should be consistent with them.